src/Snowflake/Command.php

```diff
@@ -149,18 +226,270 @@ public static function cloneDatabaseFromShared(Connection $connection, array $da
                         QueryBuilder::quoteIdentifier($schemaName),
                         QueryBuilder::quoteIdentifier($tableName),
                     ));
-                }
 
-                var_dump($schemaName);
-                if ($k > 9) {
-                    exit();
+                    foreach ($tableGrants as $tableGrant) {
+                        self::assignGrantToRole($connection, $tableGrant);
+                    }
                 }
             }
         }
     }
 
-    public static function exportUsersAndRoles(Connection $sourceSnflkConnection): array
+    public static function createRole(Connection $connection, array $role): void
```

Comment:
Tohle nějak předpokládá, že v $role je privilege OWNERSHIP? Jestli jo, tak bych sem dal assert, že to tak fakt je

Reply:
jo tak měl by tam být ownership - přidal jsem assert - https://github.com/keboola/project-migration-tool/pull/1/commits/58870a7144a34ee2e0259ab1dc7fcf5e350affac